internal/x509fork/README.md

```diff
@@ -0,0 +1,20 @@
+# x509fork
+
+This is a minimalist fork of [`crypto/x509`](https://pkg.go.dev/crypto/x509). 
+
+[!WARNING] This library is not safe to use for applications outside of this repository.
+
+[!WARNING] This fork will not be kept in synced with upstream. It will not be updated, unless required by a security vulnerability or a critical functionality issue.
```

Comment:
```suggestion
> [!WARNING] 
> This fork will not be kept in synced with upstream. It will not be updated, unless required by a security vulnerability or a critical functionality issue.
```